src/controllers/payment.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger, createAndLogErrorRequest } from "../utils/logger";
+import { CHECK_YOUR_ANSWERS_URL, CONFIRMATION_URL, PAYMENT_PAID } from "../config";
+import { ApplicationData } from "../model";
+import { getApplicationData } from "../utils/application.data";
+import { Payment, PaymentKey } from "../model/payment.type.model";
+
+// The Payment Platform will redirect the user's browser back to the `redirectUri` supplied when the payment session was created,
```

Comment:
👍  liking the comments. The whole payment process is so decoupled with redirects that these will help with maintenance.

Reply:
Thanks, following https://github.com/companieshouse/dapperdox.developer.ch.gov.uk/blob/master/assets/sections/payment-api/templates/guides/overview.md